Entities/Items/Sponge/SpongeCommon.as

```diff
@@ -0,0 +1,9 @@
+
+const int ABSORB_COUNT = 100;
+const string ABSORBED_PROP = "absorbed";
+
+
+void spongeUpdateSprite(CSprite@ this, u8 absorbed)
+{
+	this.animation.setFrameFromRatio(f32(absorbed) / ABSORB_COUNT);
```

Comment:
Something that would be nice is if the current frame is greater than the new frame (i.e the sponge has dryed by one frame) Is to make some water particles like the water is coming out of it.

Reply:
Should be in latest commit